CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+Creating a more accessible tooltip experience for users is one of
+the key goals of this proposal, so additional ideas from the
+community that should be considered to help in further improving
+the current situation is very much welcome.
+
+### Zoom
+
+Built-in tooltips today do not adhere to user preferred zoom levels,
+making them less accessible than the content they are associated with.
+
+Making these tooltips stylable through the new `::tooltip` pseudo
+element should allow for a more accessible experience in such
+scenarios.
+
+## Security Considerations
+
+As noted in the section on the [current landscape of built-in
+tootlips](#current-landscape-of-built-in-tootlips), browser built-in
+tooltips are currently allowed to escpape the bounds of the browser
+window.
+
+For tooltips styled via the `::tooltip` pseudo element, UAs **must** instead
+stay within the bounds of the web page. This would avoid [security concerns](
+https://textslashplain.com/2017/01/14/the-line-of-death/) when rendering
+outside the bounds of the browser window, which becomes exacerbated when
+styleability is involved, because this can more easily trick users into
+clicking on a spot on the screen that could lead to a security vulnerability.
+
+This behavior would be limiting for Iframes in particular but would be
+consistent with the behavior authors get with custom JS tooltips already,
+so applying the same restriction here likely wouldn’t be a major downgrade
+for authors.
+
+## Privacy Considerations
+
+There are no known privacy considerations to take into account as part of
+this proposal.
+
+## Open questions
+
+- [What styles should be allowed within the `::tooltip` pseudo
+element?](https://github.com/w3c/csswg-drafts/issues/9447)
+- Should `::tooltip` be based on `interesttarget` or should that
+detail be left up to the UA?
+- Should we consider another method for triggering `::tooltip` and its
+base styles than the `appearance` property?
+- Is `content` the best mechanism for setting the tooltip text from
+`title` within the `::tooltip` psuedo element?
+- Should authors be allowed to change the `content` of the `::tooltip`?
+- Should authors be able to change the positioning of a `::tooltip` or
+should this be magic left up to the UA?
+- What are the right default UA styles for `::tooltip`?
+- Does it make sense for `::tooltip` to be a [Tree-Abiding
+Pseudo-element](https://drafts.csswg.org/css-pseudo/#treelike)?
+- How should `::tootltip` interact with `interesttarget` and any other
+`popover` on the same element?
+- Are there other ways we improve the current accessibility landscape
+of the `title` attribute?
+- Should the text within `::tooltip` be selectable?
+- How should `::tooltip` be represented in the DOM tree?
+- Should we use `visibility` for showing the `::tooltip` transition,
+or [`position-visibility`](
+https://drafts.csswg.org/css-anchor-position-1/#position-visibility)?
+If `position-visibility`, we may want to consider adding a new value
```

Comment:
I was going to raise another issue - what happens with `pointer-events: none`, but it looks like that already disables the native tooltips in Chromium and Firefox, so this wouldn't change anything in that regard (!)

Reply:
Ack, thanks for the callout, glad to hear this shouldn't be a problem!